odoo/addons/base/models/ir_model.py

```diff
@@ -1228,20 +1228,21 @@ def check(self, model, mode='read', raise_exception=True):
             r = self._cr.fetchone()[0]
 
         if not r and raise_exception:
-            groups = '\n\t'.join('- %s' % g for g in self.group_names_with_access(model, mode))
+            descr = self.env[model]._description or model
```

Comment:
and to avoid the "Désolé, vous n’êtes pas autorisé a lire 'Users'" `self.env['ir.model']._get(self._name).name` (should this be an helper?)